qcfractal/interface/collections/openffworkflow.py

```diff
@@ -85,14 +85,14 @@ class DataModel(Collection.DataModel):
             }
         }
         torsiondrive_static_options: Dict[str, Any] = {
-            "torsiondrive_meta": {},
-            "optimization_meta": {
+            "keywords": {},
```

Comment:
This is fine, more from a user perspective, do we have anyone we need to tell about this change?

Reply:
Not yet, no.